gnome-settings/setup-gnome-tweaks.sh

```
@@ -0,0 +1,416 @@
+    
+    if [ -f "$backup_dir/mutter.dconf" ]; then
+        dconf load /org/gnome/mutter/ < "$backup_dir/mutter.dconf"
+        log_success "Mutter設定を復元しました"
+    fi
+    
+    log_success "🎉 設定の復元が完了しました！"
+}
+
+# 現在の設定をエクスポートする関数
+export_current_settings() {
+    local export_dir="gnome-settings-export-$(date +%Y%m%d_%H%M%S)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)